### PR TITLE
December 2019 Content Update

### PR DESCRIPTION
**Pull request by shirolkar**
This PR address the following two issues:
https://github.com/microsoft/MCW-Cognitive-services-and-deep-learning/issues/30
https://github.com/microsoft/MCW-Cognitive-services-and-deep-learning/issues/31
Thanks.
Jitendra

### CODE REVIEW

**Comment by DawnmarieDesJardins** [comment]
@shirolkar 
Hi Jitendra!  Please see PR #33 - it's feedback to this PR from one of our SMEs.
Thanks, Dawnmarie

**Comment by shirolkar** [comment]
> @shirolkar
> Hi Jitendra! Please see PR #33 - it's feedback to this PR from one of our SMEs.
> Thanks, Dawnmarie

@DawnmarieDesJardins @timahenning Hi Dawnmarie - the changes from Rodrigo are good. We can merge his PR. Thanks.

File: Hands-on lab/HOL step-by step - Cognitive services and deep learning.md [hunk 1]
```diff
@@ -62,46 +62,45 @@ In this workshop, you will help Contoso Ltd. build a proof of concept that shows
 
 The high-level architecture of the solution is illustrated in the diagram. The lab is performed within the context of a notebook running within Azure Notebooks. Various notebooks are built to test the integration with the Cognitive Services listed, to train custom ML services, and to integrate the results in a simple user interface that shows the result of processing the claim with all of the AI services involved.
 
-![The High-level architectural solution begins with a Claim, which us submitted for processing using a notebook in Azure Databricks. This notebook coordinates the calls to Computer Vision, Text Analytics, and Containerized Services, which includes a Classification Service and a Summary Service that both process claim text.](media/image2.jpg 'High-level architectural solution')
+![The High-level architectural solution begins with a Claim, which us submitted for processing using a notebook in Azure Databricks. This notebook coordinates the calls to Computer Vision, Text Analytics, and Containerized Services, which includes a Classification Service and a Summary Service that both processes claim text.](media/image2.jpg 'High-level architectural solution')
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/HOL step-by step - Cognitive services and deep learning.md]: Please review the code path to all of the images - I don't believe these are full links.  Shortened links cause images in the HTML files to not show up.

File: Whiteboard design session/WDS student guide - Cognitive services and deep learning.md [hunk 2]
```diff
@@ -130,112 +130,124 @@ Timeframe: 60 minutes
...
 
 _Summarizing claim text_
 
-1.  The team at Contoso has heard about a Python library called gensim that has a summarize function. Given an input of text, it can extract a summary of the desired length. Contoso would like their PoC to implement its summarization functionality initially using gensim. However, the process they follow to deploy the summarization capability should also enable them to replace the use of gensim with another library or with the use of their own custom trained models if desired down the road. Describe how Contoso should deploy the summarization service to meet these requirements?
+1. The team at Contoso has heard about a Python library called gensim that has a summarize function. Given an input of text, it can extract a summary of the desired length. Contoso would like their PoC to implement its summarization functionality initially using gensim. However, the process they follow to deploy the summarization capability should also enable them to replace the use of gensim with another library or with the use of their own custom trained models if desired down the road. Describe how Contoso should deploy the summarization service to meet these requirements?
```
Comment (DawnmarieDesJardins) [inline Whiteboard design session/WDS student guide - Cognitive services and deep learning.md]: Shouldn't all instances of Gensim be capitalized?  Like Keras and TensorFlow.

Comment (shirolkar) [reply Whiteboard design session/WDS student guide - Cognitive services and deep learning.md]: @DawnmarieDesJardins @timahenning I have capitalized Gensim in both student and trainer guides including the presentation - those cover all the locations.

File: Hands-on lab/HOL step-by step - Cognitive services and deep learning.md (hunk 1 above)
Comment (shirolkar) [reply Hands-on lab/HOL step-by step - Cognitive services and deep learning.md]: @DawnmarieDesJardins @timahenning I have reviewed all image paths and made them to absolute paths as described in the issue https://github.com/shd101wyy/markdown-preview-enhanced/issues/34.

Comment (DawnmarieDesJardins) [reply Hands-on lab/HOL step-by step - Cognitive services and deep learning.md]: Absolute paths!  Thank you Jitendra...I couldn't remember the correct term. :)

**Pull request merged**